style.css

```diff
@@ -66,6 +69,98 @@ form input {
     fill: var(--red);
 }
 
+
+/* main card */
+
+main {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-grow: 1;
+}
+
+
+p {
+    margin: 1rem;
+}
+
```

Comment:
You have a lot of empty rows in this CSS document. Let's delete the extra empty rows and have just one, between rules.

Example:

```css
.some-class {
  /* ... */
}

.some-other-class {
  /* ... */
}
```